tests/js/spec/views/performance/transactionDetails/index.spec.jsx

```diff
@@ -0,0 +1,50 @@
+import {cleanup, mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import ProjectsStore from 'app/stores/projectsStore';
+import EventDetails from 'app/views/performance/transactionDetails';
+
+describe('EventDetails', () => {
+  afterEach(cleanup);
+
+  it('renders alert for sample transaction', () => {
+    const project = TestStubs.Project();
+    ProjectsStore.loadInitialData([project]);
+    const organization = TestStubs.Organization({
+      features: ['performance-view'],
+      projects: [project],
+    });
+    const event = TestStubs.Event();
+    const routerContext = TestStubs.routerContext([]);
+
+    const {queryByText} = mountWithTheme(
+      <EventDetails
+        organization={organization}
+        params={{orgId: organization.slug, eventSlug: `${project.slug}:${event.id}`}}
+        location={routerContext.context.location}
+      />,
+      {context: routerContext}
+    );
+    expect(queryByText('Get Started')).toBeTruthy();
```

Comment:
Can we check if the alert exists by using the `FinishSetupAlert` or `AlertBar` selector? I don't feel safe doing this - what if somebody changes the `EventDetails`  mock data and "Get Started" showed up somewhere else?

Reply:
@Neo-Zhixing Actually no, I don't think it's possible to check if that component exists using React Testing Library (as opposed to Enzyme). But I changed the text to search for that's definitely unique.